src/compliance/trusted-claim-issuers.service.ts

```diff
@@ -0,0 +1,69 @@
+import { Injectable } from '@nestjs/common';
+import {
+  Asset,
+  ModifyAssetTrustedClaimIssuersAddSetParams,
+  ModifyAssetTrustedClaimIssuersRemoveParams,
+  TrustedClaimIssuer,
+} from '@polymeshassociation/polymesh-sdk/types';
+
+import { AssetsService } from '~/assets/assets.service';
+import { ServiceReturn } from '~/common/utils';
+import { RemoveTrustedClaimIssuersDto } from '~/compliance/dto/remove-trusted-claim-issuers.dto';
+import { SetTrustedClaimIssuersDto } from '~/compliance/dto/set-trusted-claim-issuers.dto';
+import { TransactionsService } from '~/transactions/transactions.service';
+
+@Injectable()
+export class TrustedClaimIssuersService {
+  constructor(
+    private readonly assetsService: AssetsService,
+    private readonly transactionsService: TransactionsService
+  ) {}
+
+  public async find(ticker: string): Promise<TrustedClaimIssuer<true>[]> {
+    const asset = await this.assetsService.findOne(ticker);
+
+    return asset.compliance.trustedClaimIssuers.get();
+  }
+
+  public async set(ticker: string, params: SetTrustedClaimIssuersDto): ServiceReturn<Asset> {
+    const { signer, webhookUrl, ...rest } = params;
+    const asset = await this.assetsService.findOne(ticker);
+
+    return this.transactionsService.submit(
+      asset.compliance.trustedClaimIssuers.set,
+      rest as ModifyAssetTrustedClaimIssuersAddSetParams,
```

Comment:
```suggestion
      rest,
```
If the DTO matches the signature there is no need to assert this.

The SDK is checking explicitly for null in a function, so this assertion is unsound.

`(aTrustedFor === null && bTrustedFor === null)`
https://github.com/PolymathNetwork/polymesh-sdk/blob/339b750301f99ccff0c390a00fdbf1f153765999/src/api/procedures/modifyAssetTrustedClaimIssuers.ts#L84